Eplant/views/ChromosomeViewer/Viewer/Viewer.tsx

```diff
@@ -0,0 +1,52 @@
+// -------
+// IMPORTS
+// -------
+
+import { FC } from 'react'
+
+import Typography from '@mui/material/Typography'
+
+import { ChromosomeItem } from '../types'
+
+import Chromosome from './Chromosome'
+
+// TYPES
+interface ViewerProps {
+  chromosomes: ChromosomeItem[]
+  scale: number
+}
+//----------
+// COMPONENT
+//----------
+const Viewer: FC<ViewerProps> = ({ chromosomes, scale }) => {
+  return (
+    <div
```

Comment:
nit: If possible try and use MUI boxes to keep things consistent

Reply:
using Box on line 23 messes up the spacing of the chromosomes